karma.conf.js

```diff
@@ -23,7 +23,18 @@ var staticConfig = {
   concurrency: Infinity,
   browserDisconnectTimeout: 300000, // 5 minutes
   browserNoActivityTimeout: 300000,
-  captureTimeout: 300000
+  captureTimeout: 300000,
+  customLaunchers: {
+    PhantomJSDebug: {
```

Comment:
Changing the browser @ line 21 to `PhantomJSDebug` will pause the test run for 15s and listen for a remote debugger session to open.